wsf_scraping/items.py

```diff
@@ -4,6 +4,7 @@
 
 class BaseArticle(scrapy.Item):
     title = scrapy.Field()
+    year = scrapy.Field()
```

Comment:
if there is no year metadata?

Reply:
We'll return a null year, but it's better than no year attribute, as it's something we want to scrape more often

Reply:
👍